RELEASING.md

```diff
@@ -0,0 +1,21 @@
+## Release checklist
+
+- [ ] Branch off of `develop` and name the branch `version-X.X.X`
+- [ ] Update version number within the repository
+  - in `setup.py`
+  - in Copyright Notice in `README`
+  - in the `LICENSE`
+- [ ] Update classifiers in `setup.py`
+- [ ] Update `HISTORY` with a list of changes for this version
+- [ ] Verify that all tests pass (`python setup.py test`)
```

Comment:
While the code should follow style guide all the time, I think it should be checked (`pylint` and `flake8`) here in particular. Also spelling might make sense to be checked here again. Just to make sure we have a clean release.

Reply:
Fair point. If these checks are done upstream (perhaps included in continuous integration) then it shouldn't need to be done here. But might be good to check again.

Reply:
By the way, we need to do a fair amount of work just to get the current code up to snuff with `pylint`.

Reply:
I moved this discussion to #125. Maybe just mention it in RELEASING might still be worth it, it is easy to cross off a box as long as there are not too many yet.